SimpleToDo/SimpleToDo/ToDoItem.swift

```diff
@@ -48,6 +48,22 @@ struct ToDoItem: Codable, Identifiable, Hashable {
         }
     }
 
+    /// The value of the item, to convey this information to assistive technologies.
+    var accessibilityValue: String {
+        if isComplete {
+            return "completed"
+        } else {
+            switch priority {
+            case .low:
+                return "low priority"
+            case .medium:
+                return "medium priority"
```

Comment:
Wasn't sure if it would be OK to not add anything related to priority here, but that would make this return `String?` rather than `String` which I felt would make the place where this is used harder to understand (`accessibilityValue(_:)` does not handle a `String?` in a straightforward way)

Reply:
Is it considered bad practice to include an empty accessibility value for that instance? I mean effectively putting `""` in there for medium priority, to avoid clutter.

Reply:
It's not entirely clear from the documentation if this has any side effects, so good idea. 👍